src/modules/iasql_functions/iasql.ts

```diff
@@ -322,6 +323,22 @@ ${Object.keys(tableCollisions)
         dbId,
       );
     }
+    (async () => {
+      const user = await MetadataRepo.getUserFromDbId(dbId);
+      if (user && (moduleNames.length > 0 || moduleNames[0] !== 'aws_account')) {
+        // ignore installing of aws_account only
+        telemetry.logInstall(
+          user?.id,
+          {
+            dbId,
+          },
+          {
+            params: moduleNames,
+            output: 'Done!',
+          },
+        );
+      }
+    })();
```

Comment:
BTW this being non-blocking and run asynchronously is ok, right?

Reply:
I explicitly asked for this. Metrics shouldn't ever block the actual usage for the users. We don't care if they're a fraction of a second late or whatever.